examples/tutorial/jupyter/execution/pandas_on_ray/README.md

```diff
@@ -0,0 +1,19 @@
+# Jupyter notebook examples to run Modin on PandasOnRay
+
+## Creating a development environment
+
+To get required dependencies for these Jupyter Notebooks
+you should create a development environment with `pip`
+using `requirements.txt` file located in the current directory:
+
+```bash
+pip install -r requirements.txt
```

Comment:
We have lately had a hard time with this. Sometimes pip is installing every version. Should we add a comment in this document about manually installing each line in case pip fails?

Reply:
Yes, I think we should add a note on this.

Reply:
Rebased on master and added a note.